src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py

```diff
@@ -11,55 +11,55 @@
 MAPPINGS = [
     ProductMapping(re.compile(r"(TRUE|FREE)NAS-MINI-3.0-E$"), [
         VersionMapping(re.compile(".*"), [
-            MappingSlot(0, 0, False),
             MappingSlot(0, 1, False),
             MappingSlot(0, 2, False),
             MappingSlot(0, 3, False),
-            MappingSlot(0, 5, False),
             MappingSlot(0, 4, False),
+            MappingSlot(0, 5, False),
+            MappingSlot(0, 6, False),
         ]),
     ]),
     ProductMapping(re.compile(r"(TRUE|FREE)NAS-MINI-3.0-E\+$"), [
         VersionMapping(re.compile(".*"), [
-            MappingSlot(0, 0, False),
             MappingSlot(0, 1, False),
             MappingSlot(0, 2, False),
             MappingSlot(0, 3, False),
-            MappingSlot(0, 0, False),
+            MappingSlot(0, 4, False),
             MappingSlot(0, 1, False),
+            MappingSlot(0, 2, False),
```

Comment:
Doesn't this just repeat line 25 ? it seems like line 28 also repeats line 24 🤔

Reply:
It does, good catch

Reply:
Actually, this makes it so that TrueNAS MINI E enclosure mgmt probably hasn't worked the way it's supposed to have 🙂

Reply:
`for i in ${UNTIL_CORRECT}; do
    repeat code` 😜